### ISSUE TITLE
Quantization leads to performance degradation

### ISSUE THREAD

**Issue by baoguo1995** [OpenNMT/CTranslate2#1273]
When converting the model, I enable the quantization to 'int8', but I noticed a decrease in performance of the converted model by 5 points in terms of BLEU. 
Therefore, I would like to inquire if there are any specific considerations I should be aware of when performing quantization.

command: ct2-fairseq-converter --model_path .  --quantization int8  -output_dir .

Environment:
ctranslate2=3.14.0
fairseq=0.10.2

**Comment by guillaumekln** [comment]
Quantization can lead to a performance degradation but it is generally very small for a standard encoder-decoder Transformer (less than 1 BLEU).

Are you comparing the BLEU score against Fairseq or against the CTranslate2 model without quantization? Note that CTranslate2 uses a different default beam size than Fairseq: 

https://opennmt.net/CTranslate2/guides/fairseq.html#beam-search-equivalence

**Comment by baoguo1995** [comment]
> Quantization can lead to a performance degradation but it is generally very small for a standard encoder-decoder Transformer (less than 1 BLEU).
> 
> Are you comparing the BLEU score against Fairseq or against the CTranslate2 model without quantization? Note that CTranslate2 uses a different default beam size than Fairseq:
> 
> https://opennmt.net/CTranslate2/guides/fairseq.html#beam-search-equivalence

Yes, I have compard the BLEU score of Fairseq model against the CTranslate2 model with quantization float16 and found that the results are satisfactory(less than 1 BLEU).
